addons/medical_treatment/stringtable.xml

```diff
@@ -3496,5 +3496,13 @@
             <Chinesesimp>个人急救包使用条件</Chinesesimp>
             <Chinese>個人急救包使用條件</Chinese>
         </Key>
+        <Key ID="STR_ACE_Medical_Treatment_AllowSelfStitch_DisplayName">
```

Comment:
Move strings to top with the other (new) settings, after L217. I guess these are left over entries that need to cleaned up eventually.

Reply:
Done